restapi/restapi.go

```diff
@@ -292,3 +339,111 @@ func (r *RestAPI) getSalesforceOauthToken() (string, error) {
 
 	return authResponse.AccessToken, nil
 }
+
+func (r *RestAPI) setDefaults() {
+	if r.ListMethod == "" {
+		r.ListMethod = r.Method
+	}
+	if r.ListMethod == "" {
```

Comment:
This is the same `if` statement as the one above.

Reply:
Yes, it looks strange, so I added some comments.

Reply:
Sorry. I hadn't looked inside the blocks carefully.

Reply:
No problem!